start.sh

```diff
@@ -6,6 +6,7 @@ function abort() {
 
 set -mx
 
+mkdir -p /etc/guacamole
```

Comment:
Note: the container is running under user `guacd`, it may not have permission to create that directory.

Reply:
This has been tested and it works.